lib/components/Logos/styles.ts

```diff
@@ -3,6 +3,7 @@ import breakpoints from "../../theme/breakpoints";
 
 export const logoWithClaim = css`
   max-width: 15rem;
+  padding-right: 1rem;
```

Comment:
```suggestion
  padding-inline-end: 1rem;
```
as a habit, always avoid `right` or `left` so that the layout should work when we enable an RTL language